assets/prefabs/structures/Saw.prefab

```diff
@@ -0,0 +1,809 @@
+{
+  "StructureTemplate" : {
+    "type": "GooeysQuests:Saw",
```

Comment:
There is no Saw type yet, and no structure spawns structures of that type. 

Have a look at the prefabs in the folder `assets/prefabs/structures/types`.